node/src/validator/mod.rs

```diff
@@ -258,8 +258,9 @@ impl<N: Network, C: ConsensusStorage<N>> Validator<N, C> {
 
         // Load the primary's keys.
         let primary_key_file = format!("{base_path}.primary-{primary_id}-key.json");
-        let primary_keypair =
-            read_authority_keypair_from_file(primary_key_file).expect("Failed to load the node's primary keypair");
+        let primary_keypair = read_authority_keypair_from_file(primary_key_file).unwrap_or_else(|_| {
+            panic!("Failed to load the node's primary keypair {base_path}.primary-{primary_id}-key.json")
```

Comment:
why not add the actual error?

Reply:
Oh good point, I was using `except(format!(...))` before and just blindly took clippy's advice for this replacement. Will add the error logging too.